digitalocean/app/app_spec.go

```diff
@@ -189,6 +188,40 @@ func appSpecAppLevelAlerts() *schema.Resource {
 				Default:  false,
 				Optional: true,
 			},
+			"notifications": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"email": {
```

Comment:
For consistency with HTTP API

```suggestion
						"emails": {
```

Reply:
It's not like this. I missed our api with alert team's api. They use these.